challenges/classes.js

```diff
@@ -1,7 +1,48 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+function CuboidMaker(length, width, height) {
+    this.length = length
+    this.width = width
+    this.height = height
+}
+
+
+CuboidMaker.prototype.volume = function () {
+    return this.length * this.width * this.height
+}
+class volume {
+    constructor(length, width, height) {
+        this.length = length;
+        this.width = width;
+        this.height = height;
+    }
+
+    get volume() {
+        return this.calcArea();
+    }
+    calcArea() {
+        return this.length * this.width * this.height;
+    }
+}
+
+const square = new volume(4, 5, 5);
+
+
+CuboidMaker.prototype.surfaceArea = function () {
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+}
+class surfaceArea {
+    constructor(length, width, height) {
+        this.length = length;
+        this.width = width;
+        this.height = height;
+    }   
+}
+
+cuboid = new CuboidMaker(4, 5, 5)
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
+console.log(cuboid.surfaceArea()); // 130
```

Comment:
Let's rework this one. They are asking to take your constructor function, called CuboidMaker...and refactor it into _class CuboidMaker_. The methods on CuboidMaker are supposed to be _volume_ and _surafaceArea_